### PR TITLE
Update wiki links

### PR DESCRIPTION
**Pull request by dyfer**
As a follow-up to #247 I've made following changes:

- in the top navigation bar I replaced Wiki link from "Pages" to SuperCollider wiki (*)
- on the main page, I updated the link to the Wiki so that it points to SuperCollider wiki
  - I've also removed links to the old wikis, as they are now linked from the main SuperCollider wiki

(*) I'm not fluent in Jekyll and I'm not sure if I created the "external link" correctly - it seems hack-y... I've added a new layout that refreshes the page, pointing to the external URL. Is there a better way to add an external link to navigation?

### CODE REVIEW

File: _layouts/external.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```
Comment (gusano) [inline _layouts/external.html:1]: The new layout is not needed, see below

File: wiki.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (gusano) [inline wiki.html:1]: This is also overkill IMHO

**Review by gusano — CHANGES_REQUESTED** [review]
@dyfer Thank you!
This jekyll setup is very messy (to say the least) so it's not easy to understand how to change things (I'm working on a new website and will clean all that hopefully soon).

A cleaner way to change the WIKI link would be to:

- remove `pages.html` (this will remove the automatic *Wiki* entry in the nav, you did it already)
- do not add any new layout
- do not add a new `wiki.html` page
- add the new *Wiki* link directly to `_layout/default.html` line 57, just before the link to the Docs, as this won't be automatically generated

Again, the current thing is really messy and there are a lot of unused pages/layouts/etc which makes things a bit difficult to understand ;)

**Comment by dyfer** [comment]
Thanks @gusano ! This was super helpful, I think this is better now.

**Review by gusano — APPROVED** [review]
Thank you!

**Pull request merged**